scripts/ci/ipxe-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+mkdir ipxe
+cp build/tmp/deploy/images/genericx86-64/dts-base-image-genericx86-64.cpio.gz ipxe
+cp build/tmp/deploy/images/genericx86-64/bzImage ipxe
```

Comment:
* Maybe don't hardcode paths but use arguments?
* You could create ipxe dir inside open-source-firmware-validation and add it to `.gitignore` so we don't push it by mistake. You could use: <https://stackoverflow.com/a/246128> to always get correct path and go from there
* Not sure if needed but you could probably use symlinks instead of copying.
* Don't start detached, maybe leave it to user, if they want then they can just add `&` when running script.

Reply:
- Changed to symlinks
- Now you can add 2 files (dts-base-image and BzImage)
- Removed detached mode by default
- ipxe directory is ignored by .gitignore